Update citations.md (#685)
JuliaReach/Reachability.jl: docs/src/citations.md

## Conference Proceedings

> - Bogomolov, S., Frehse, G., Gurung, A., Li, D., Martius, G., & Ray, R. (2019). *Falsification of hybrid systems using symbolic reachability and trajectory splicing.* In Proceedings of the 22nd ACM International Conference on Hybrid Systems: Computation and Control (pp. 1-10). ACM.

Reviewer: ```suggestion
- Bogomolov, S., Frehse, G., Gurung, A., Li, D., Martius, G., & Ray, R. (2019). *Falsification of hybrid systems using symbolic reachability and trajectory splicing.* In Proceedings of [HSCC'19](http://hscc2019.eecs.umich.edu/): 22nd ACM International Conference on Hybrid Systems: Computation and Control (pp. 1-10). ACM.
```
(to be consistent with the next reference)

- Bak, Stanley, Hoang-Dung Tran, and Taylor T. Johnson. *Numerical Verification of Affine Systems with up to a Billion Dimensions.* arXiv preprint arXiv:1804.01583 (2018). Accepted in Proceedings of [HSCC'19](http://hscc2019.eecs.umich.edu/): 22nd ACM International Conference on Hybrid Systems: Computation and Control (HSCC'19).

- Schupp, Stefan, and Erika Ábrahám. *Spread the Work: Multi-threaded Safety Analysis for Hybrid Systems.* International Conference on Software Engineering and Formal Methods. Springer, Cham, 2018.

- Schupp, Stefan, Justin Winkens, and Erika Ábrahám. *Context-Dependent Reachability Analysis for Hybrid Systems.* 2018 IEEE International Conference on Information Reuse and Integration (IRI). IEEE, 2018.

## Journals

> - Althoff, M. (2019). *Reachability analysis of large linear systems with uncertain inputs in the krylov subspace.* IEEE Transactions on Automatic Control.

Reviewer: ```suggestion
- Althoff, M. (2019). *Reachability analysis of large linear systems with uncertain inputs in the Krylov subspace.* IEEE Transactions on Automatic Control.
```

## Preprints

- Roehm, H., Oehlerking, J., Woehrle, M., & Althoff, M. (2019). *Model Conformance for Cyber-Physical Systems: A Survey.* ACM Transactions on Cyber-Physical Systems, 3(3), 30.

- Minghui, S., Bakirtzis, G., Jafarzadeh, H., & Fleming, C. (2019). *Correct-by-construction: a contract-based semi-automated requirement decomposition process.* arXiv preprint arXiv:1909.02070.

- Liu, C., Arnon, T., Lazarus, C., Barrett, C., & Kochenderfer, M. J. (2019). *Algorithms for Verifying Deep Neural Networks.* arXiv preprint arXiv:1903.06758.

- Julian, K. D., & Kochenderfer, M. J. (2019). *A Reachability Method for Verifying Dynamical Systems with Deep Neural Network Controllers.* arXiv preprint arXiv:1903.00520.

- Mitchell, Ian M., Jacob Budzis, and Andriy Bolyachevets. *Invariant, Viability and Discriminating Kernel Under-Approximation via Zonotope Scaling.* arXiv preprint arXiv:1901.01006 (2019).

> - Dang, T. (2019). *Reachability Analysis and Hybrid Systems Biology-In Memoriam Oded Maler.* In International Workshop on Hybrid Systems Biology (pp. 16-29). Springer, Cham.

Reviewer: ```suggestion
- Dang, T. (2019). *Reachability Analysis and Hybrid Systems Biology - In Memoriam Oded Maler.* In International Workshop on Hybrid Systems Biology (pp. 16-29). Springer, Cham.
```
